### ISSUE TITLE
typo in Aluminium tables

### ISSUE THREAD

**Issue by hightower8083** [ComputationalRadiationPhysics/picongpu#2864]
When defining the ionizable `Al` ions, there is some confusion in the tables:
- in `ionization::atomicNumbers` it is written as `Aluminium_t`
- in `ionization::effectiveNuclearCharge` it is `Aluminum` (**!**)
- in `ionization::energies::AU` it is again `Aluminium`

So to make it work I had to do:
```
    atomicNumbers< ionization::atomicNumbers::Aluminium_t >,
    ionizationEnergies< ionization::energies::AU::Aluminium_t >,
    effectiveNuclearCharge< ionization::effectiveNuclearCharge::Aluminum_t >,
```

I suggest sticking to  a more european and historical `Aluminium` spelling everywhere.

**Comment by n01r** [comment]
Thanks for spotting this inconsistency! :+1:

**Comment by ax3l** [comment]
fixed with #2865 :)
